app/src/main/res/drawable/ic_backspace.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.ic_backspace` appears to be unused
 Unused resources make applications larger and slow down builds.

Reply:
I will use it in the next pull request